### ISSUE TITLE
Whitespace misplaced for comments following "run:"

### ISSUE THREAD

**Issue by ashwinvis** [snakemake/snakefmt#61]
For this simple file:

```py
# hello world
rule hello:
    run:
        print("hello world")


# yet another hello world
rule hello2:
    shell:
        """echo hello world"""


# final one
rule hello3:
    shell:
        """printf 'hello world'"""
```

The following diff is obtained:
```diff
❯ snakefmt test.smk --diff
=====> Diff for test.smk <=====

  # hello world
  rule hello:
      run:
          print("hello world")
  
  
- # yet another hello world
+         # yet another hello world
? ++++++++
+ 
+ 
  rule hello2:
      shell:
          """echo hello world"""
  
  
  # final one
  rule hello3:
      shell:
          """printf 'hello world'"""

[INFO] All done 🎉
```

**Comment by mbhall88** [comment]
Interesting. Thanks for reporting @ashwinvis. Could you please confirm which version you are using? We just released a new patch version today (0.1.4). I don't know if that will fix this issue, but best to confirm that first.

**Comment by ashwinvis** [comment]
The latest and the greatest.
```
❯ pip list | rg snake    
snakefmt                      0.1.4
snakemake                     5.23.0
```

I wonder if it is a bug with snakemake in how it groups Pythonic stuff after a rule, ignoring indentation. I changed "yet another hello world" to a print statement and it seems to be executed if I call `snakemake -s test.smk hello -j`.

**Comment by bricoletc** [comment]
Hi @ashwinvis, thanks for spotting this, it is definitely not a snakemake bug. It's actually tied to an annoyance in python's [tokenize](https://docs.python.org/3/library/tokenize.html) specifically related to comments. Will have a look at this, not the easiest fix!

**Comment by bricoletc** [comment]
This is now addressed 9402a48
